### DISCUSSION TITLE
Please add option to disable new version notification

### DISCUSSION THREAD

**Discussion by s101d1** [Genymobile/feedback#49] [Genymotion Desktop] [answered]
As the title says, please add option to disable new version download notification, I found the notification annoying.

**Comment by agavignet** [comment, marked as answer]
Sorry for the late answer, I understand your point but even if we're a small team, we regularly release new versions to fix bugs, add new features and improve the reliabiity. We want users to upgrade to latest version to enjoy an improved user experience as soon as possible. That's why we keep your feature request for the future but this is a very low priority to us.

Thanks for your understanding,
Adrien

**Reply by s101d1** [reply]
How about moving the pop-up notification to somewhere else for now? The main problem with the notification is it's intrusive, it blocks and covers the Run device button (see the screenshot image below)

![gm1](https://github.com/Genymobile/feedback/assets/8465087/1587d484-9057-4372-b745-26410c957eed)
